hash/msx_softcard.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0
+-->
+<softwarelist name="msx_softcard" description="MSX1 Astron SoftCards">
+
+<!--
+Known releases:
+*5302    Shark Hunter (UK)
+5304    Backgammon (UK)
+5305    Le Mans 2 (UK)
+5306/1  The Wreck (UK)
+*5307    Barnstormer (UK)
+*5310    Sweet Acorn (UK)
+*5311    Chack'n Pop (UK)
+*5312    ChoroQ (UK)
+*5313    Xyzolog (UK)
+*DB014   Front Line (UK)
+*DD150   Graphics Editor (UK)
+*?????   Unknown title, CAD with digitizing for MSX2 (UK)
+*5509    Quiniela Fútbolistica 1X2 (Spain)
+*6202    Backgammon (Spain)
+*6203    La Bellota (Spain)
+*????    Lotería Primitiva (Spain)
+*????    Quiniela Hipica (Spain)
+
+No dump known for entries marked with a *
+-->
+
+	<software name="backgamm">
+		<description>Backgammon (UK)</description>
+		<year>1984</year>
+		<publisher>Electric Software</publisher>
+		<info name="serial" value="5304" />
+		<part name="cart" interface="softcard">
+			<dataarea name="rom" size="32768">
+				<rom name="backgammon (europe).rom" size="32768" crc="b3694e5a" sha1="363b5a8c94cdb63a7ee72b9d25f7b5053cab319a" offset="0" />
```

Comment:
Did you want to leave the offsets in this file? You've removed them from the other new list.

Reply:
Thanks, those offsets should have been removed indeed. They are gone now.